### ISSUE TITLE
Function calls wrong color in Python

### ISSUE THREAD

**Issue by bernharl** [patstockwell/vim-monokai-tasty#29]
Hi. My function calls in Python are green and not light blue (like in SublimeText). I'm using the vim-polyglot package and have enabled let g:python_highlight_all = 1. Is this just because your color scheme doesn't work with that syntax plugin? Everything else looks good.
![Screenshot from 2020-07-28 10-01-33](https://user-images.githubusercontent.com/42568221/88636673-878e5280-d0b9-11ea-87cd-38ec9a187875.png)

**Comment by patstockwell** [comment]
Hi @bernharl. Thanks for taking the time to help improve the colorscheme 🎉 . I've added a rule to highlight the function call the same way Sublime Text does (light blue). You can see the changes I made in this commit: https://github.com/patstockwell/vim-monokai-tasty/commit/6e23360fba31a4992c292aaf8e4fd43c9f094e90

To get the update, just pull the latest changes using your package manager. If you use _Plug_ the command is `:PlugUpdate`.

Please let me know if you find any other glaring differences, I don't write that much Python so I'm happy to take suggestions. Thanks again!

**Comment by patstockwell** [comment]
Also, for me, using the standard python syntax file in neovim, Python functions are identified by the syntax group as `pythonBuiltin`. The change I made updated this group to light blue.

For reference, if you want to know which highlight group is being used for a piece of syntax, you can use the following snippet. I keep this in my _.vimrc_ to help me identify named syntax groups.

```vim
" If you don't know what the name of a particular hightlight is, you can use
" `What`. It will print out the syntax group that the cursor is currently above.
" from https://www.reddit.com/r/vim/comments/6z4aau/how_to_stop_vim_from_autohighlighting_italics_in/
command! What echo synIDattr(synID(line('.'), col('.'), 1), 'name')
```

**Comment by bernharl** [comment]
Thanks for the quick reply! It seems that the syntax package I use has different names for function calls. For it to work for me I would need the color to be defined for `pythonFunctionCall` and `pythonBuiltinFunc`. Thank you for that command by the way, very useful.

**Comment by patstockwell** [comment]
Cool, I checked vim-polyglot and it uses [vim-python](https://github.com/vim-python/python-syntax). I added the plugin and checked the named groups and I got the same names as you suggested. I've added the new rules and pushed the update 👍.

One of the differences between python syntax highlighting in _Sublime_ vs the _vim-monokai-tasty_ version is the `def` keyword. Unfortunately, the vim python syntax file lumps the `def` keyword in the same group as the `return` keyword. So if `def` is highlighted as italic light blue, then that makes `return` italic light blue too. So I've opted to leave them both magenta.

Try it out and lemme know how it goes 😄 

_example comparison:_

<img width="1269" alt="Screen Shot 2020-07-29 at 7 42 24 pm" src="https://user-images.githubusercontent.com/14284552/88784714-b3056000-d1d3-11ea-921d-7b46f3fe1046.png">
